### PR TITLE
doc: initial version of sec announcement process

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by mhdawson** [comment]
Please review/comment
@nodejs/security-wg 
@nodejs/security 
@nodejs/ctc

File: processes/security_annoucement_process.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+The Node.js community follows a process to create/review and
+then publish vulnerability announcements.  It is most often a 2 step
```
Comment (lirantal) [inline processes/security_annoucement_process.md]: extra space

File: processes/security_annoucement_process.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+process where we:
+
+* announce that releases will be made to fix an embargoed vulnerability 
+* announce that the releases with the fixes are available
+
+The process is as follows:
+
+* Security vulnerabilties are initial discussed/reviewed in the private
```
Comment (lirantal) [inline processes/security_annoucement_process.md]: initial => initially

File: processes/security_annoucement_process.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+* In original PR for the security repository for the issue, post candidate
+  text for updates that will go out with releases that will indicates 
+  releases are available and include full vulnerability details.
+
+* Once releases are made, post response to original message in
+  https://groups.google.com/forum/#!forum/nodejs-sec indicating
+  releases are available and with the full vulnerability details.
+
+* Update the blog post in
+  https://github.com/nodejs/nodejs.org/tree/master/locale/en/blog/vulnerability
```
Comment (rvagg) [inline processes/security_annoucement_process.md]: move link up by 3 paragraphs where you first talk about nodejs.org

File: processes/security_annoucement_process.md (hunk 1 above)
Comment (mhdawson) [inline processes/security_annoucement_process.md]: Fixed

File: processes/security_annoucement_process.md (hunk 2 above)
Comment (mhdawson) [inline processes/security_annoucement_process.md]: Fixed

File: processes/security_annoucement_process.md (hunk 3 above)
Comment (mhdawson) [inline processes/security_annoucement_process.md]: dONE

**Comment by mhdawson** [comment]
In terms of requesting a CVE I think that should go in a more general process for handling security vulns.  That overall process would reference processes for each of the steps including requesting CVE, doing the announcement, how to build/test security releases etc.

So just a long way to say I think it goes somewhere else.

**Comment by mhdawson** [comment]
Landed as 9e353a219d9d2858659d30446fb65dda1a91cadd

**Pull request closed without merging**